src/main/java/org/assertj/core/api/ObjectAssert.java

```diff
@@ -47,4 +48,15 @@ public ObjectAssert(AtomicReference<ACTUAL> actual) {
     return super.extracting(extractors);
   }
 
+  @Override
+  public ComparatorBasedObjectAssert<ACTUAL> usingComparator(Comparator<? super ACTUAL> customComparator) {
+    return usingComparator(customComparator, null);
```

Comment:
Minor point: should we try to describe the customComparator here?
if it is defined as a class we could pass the class name (eg: `AbsComparator`) and leave it to null if it's defined as a lambda or an anonymous class.